nss-ice-phantoms/WorkforceManagement
# Training programs -  view list and create

# Description
Added ability to view list of upcoming training programs and to create a new program.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout meg-training-list`
1. Run BangazonWorkforce
1. Verify that you have Training Programs in your database and take note of each EndDate.
1. Insert new Training Program into the database with a date in the future.
   ```
   INSERT into TrainingProgram ([Name], StartDate, EndDate, MaxAttendees) 
   VALUES ('Testing All The Things', '14 June 2019', '15 June 2019', 10)
   ```
1. In browser, go to Training Programs and verify that only upcoming programs display.
1. Click on "Create Training Program" and verify that you can create a new program, and that it displays in the list once created.


# Checklist:
